optimum/intel/openvino/configuration.py

```diff
@@ -1176,3 +1174,146 @@ def to_dict(self):
         result["weight_quantization_config"] = self.weight_quantization_config.to_dict()
         result["full_quantization_config"] = self.full_quantization_config.to_dict()
         return result
+
+
+class OVPipelineQuantizationConfig(OVQuantizationConfigBase):
+    def __init__(
+        self,
+        pipeline_quantization_configs: Dict[str, Union[Dict, OVQuantizationConfigBase]],
```

Comment:
would makes more sense to call this argument just quantization_configs because the PR already introduced something called "pipeline quantization config" so it's a bit confusing

Reply:
In my opinion `quantization_configs` is less clear when initializing config as a dictionary, but I agree that `pipeline` part seems redundant. Removed.